pages/platform/ai/notebook_tuto_11_yolov7/guide.en-gb.md

```diff
@@ -0,0 +1,118 @@
+
+![image](images/overview-notebook.png){.thumbnail}
+
+It is based on the YOLOv7 open source [repository](https://github.com/WongKinYiu/yolov7).
+
+## Requirements
+
+- Access to the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB)
+- An AI Notebooks project created inside a [Public Cloud project](https://www.ovhcloud.com/en-gb/public-cloud/) in your OVHcloud account
+- A user for AI Notebooks
+- A [Public Roboflow](https://public.roboflow.com/) account to access and download the [American Sign Language Letters Dataset](https://public.roboflow.com/object-detection/american-sign-language-letters/1)
+
+## Instructions
+
+This tutorial is based on the [American Sign Language Letters Dataset](https://public.roboflow.com/object-detection/american-sign-language-letters/1). It is available for free on [Roboflow](https://public.roboflow.com/).
+
+> *The American Sign Language Letters dataset is an object detection dataset of each ASL letter with a bounding box. [David Lee](https://www.linkedin.com/in/daviddaeshinlee/), a data scientist focused on accessibility, curated and released the dataset for public use.*
+
+> [!primary]
+>
+> To learn more about the license of this dataset, please follow this [link](https://creativecommons.org/publicdomain/zero/1.0/).
+>
+
+### Create Object Storage containers
+
+If you want to create it from the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB), go to the Object Storage section and create a new object container by clicking `Object Storage` > `Create an object container`.
+
+![image](images/new-object-container.png){.thumbnail}
+
+If you want to run it with the CLI, just follow this [guide](https://docs.ovh.com/gb/en/publiccloud/ai/cli/access-object-storage-data). You have to choose the region, the name of your container and the path where your data is located and use the following command:
+
+> [!warning]
+>
+> In this tutorial, you have to create 2 object containers.
+>
+
+- An object container `asl-volov7-data` dedicated to the dataset:
+
+```bash
+ovhai data upload <region> asl-volov7-data
+```
+
+- An object container `asl-volov7-model` to save the model weights (for a future inference for example):
+
+```bash
+ovhai data upload <region> asl-volov7-model
+```
+
+> [!warning]
+>
+> These two Object Storage containers will be empty initially. You will add data from the notebook.
+>
+
+### Launch and access Jupyter notebook with Miniconda framework
+
+You need to attach a volume if your data is in your OVHcloud Object Storage and you want to use it during your experiment, or if you need to save the results of your work in the object storage. For more information on data, volumes and permissions, see [our guide on data](https://docs.ovh.com/gb/en/publiccloud/ai/cli/access-object-storage-data/).
```

Comment:
Object Storage (1st line) 
object storage (2-3th line) 

-> Maybe Standardize this (lowercase / uppercase)